### PR TITLE
Disable systemd-timesyncd if running

### PR DESCRIPTION
**Pull request by giner**
This prevents race condition during system boot where the service which starts the latest wins.

### CODE REVIEW

**Comment by giner** [comment]
@geerlingguy would you check on this please?

**Comment by geerlingguy** [comment]
Just wondering—which OS platform(s) did you see this on? I would imagine it's maybe affecting Debian 10 Buster, but not necessarily all systemd platforms since they're not (to my knowledge) all using timesyncd?

**Comment by giner** [comment]
I experienced this issue on Ubuntu

**Comment by giner** [comment]
@geerlingguy any plans to merge or update on this?

**Comment by giner** [comment]
Friendly ping

File: tasks/main.yml [hunk 1]
```diff
@@ -20,6 +20,16 @@
   timezone:
     name: "{{ ntp_timezone }}"
 
+- name: Populate service facts
+  service_facts:
+
+- name: Disable systemd-timesyncd
```
Comment (geerlingguy) [inline tasks/main.yml:26]: This would need to be `Disable systemd-timesyncd if it's running but ntp is enabled.`

File: tasks/main.yml [hunk 2]
```diff
@@ -20,6 +20,16 @@
...
+  service:
+    name: systemd-timesyncd.service
+    enabled: false
+    state: stopped
+  when: ntp_enabled | bool and "systemd-timesyncd.service" in services
```
Comment (geerlingguy) [inline tasks/main.yml:31]: More readable:

```yaml
when:
  - ntp_enabled | bool
  - '"systemd-timesyncd.service" in services'
```

File: tasks/main.yml (hunk 1 above)
Comment (giner) [reply tasks/main.yml]: updated

File: tasks/main.yml (hunk 2 above)
Comment (giner) [reply tasks/main.yml]: updated

**Comment by giner** [comment]
Friendly ping

**Comment by kitos9112** [comment]
I'd love to see this integrated in the role as part of a sanity check 👍🏼

**Pull request merged**